kedro/framework/project/__init__.py

```diff
@@ -421,7 +420,7 @@ def find_pipelines(raise_errors: bool = False) -> dict[str, Pipeline]:  # noqa:
 
     # Handle the case that a project doesn't have a pipelines directory.
     try:
-        pipelines_package = importlib_resources.files(f"{PACKAGE_NAME}.pipelines")
+        pipelines_package = importlib.resources.files(f"{PACKAGE_NAME}.pipelines")
```

Comment:
How did this one not fail your PR? After updating my PR, now the linter complains that this does not exist.

Reply:
Oh let me look into this, I think the unit tests are passing fine and on merge to `main` we run linter on 3.11 but the linter is failing on just python 3.9 for your PR.